Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -831,6 +832,24 @@
 }
 #endif
 
+SimpleRange WebPage::getRangeSelectionContext() const
+{
+    Ref frame = CheckedRef(m_page->focusController())->focusedOrMainFrame();
+    return frame->selection().rangeByExtendingCurrentSelection(TextGranularity::ParagraphGranularity).value();
```

Comment:
Why is it okay to avoid null-checking the optional?

Reply:
```
Ref frame = CheckedRef(m_page->focusController())->focusedOrMainFrame();
```

is present in dozens of other places, and none of those places do a null check

Reply:
OH

Reply:
the optional

Reply:
oops